prime-router/settings/organizations.yml

```diff
@@ -1277,4 +1277,134 @@
         host: sftp
         port: 22
         filePath: ./upload
-        credentialName: DEFAULT-SFTP
\ No newline at end of file
+        credentialName: DEFAULT-SFTP
+- name: "vt-doh"
+  description: "Vermont Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "VT"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "vt-doh"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "classpath:/metadata/hl7_mapping/receivers/STLTs/VT/VT-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "NBS^2.16.840.1.114222.4.1.185.1^ISO"
+        receivingApplicationOID: "2.16.840.1.114222.4.1.185.1"
+        receivingFacilityName: "VDH^2.16.840.1.114222.4.1.185^ISO"
+        receivingFacilityOID: "2.16.840.1.114222.4.1.185"
+        messageProfileId: null
+        replaceValue:
+          MSH-3-1: "CDC PRIME - Atlanta,"
+        replaceValueAwithB:
+          MSH-3:
```

Comment:
We don't need this any longer for UP.  They should be included in VT-receiver-transform.yml